src/contactForm.js

```diff
@@ -77,6 +77,16 @@ class RecrasContactForm {
         return this.element.querySelectorAll(querystring);
     }
 
+    isStandalone(options) {
+        if (options.standalone) {
+            return true;
+        }
+        if (options.showSubmit) {
+            console.warn('Option "showSubmit" was renamed to "standalone". Please update your code.')
```

Comment:
I guess this message should be moved to the constructor/validator of `RecrasOptions`. That way we don't have to worry about `showSubmit` in this context at all.

Reply:
`options` does not originate from `RecrasOptions` but is only called from within `RecrasContactForm`